proxy/src/inbound.rs

```diff
@@ -128,7 +129,10 @@ mod tests {
             let srv_ctx = ctx::transport::Server::new(&ctx, &local, &remote, &Some(orig_dst));
 
             let rec = srv_ctx.orig_dst_if_not_local().map(|addr|
-                (addr, bind::Protocol::Http1(Host::NoAuthority))
+                (addr, bind::Protocol::Http1 {
+                    host: Host::NoAuthority,
+                    was_absolute_form: false,
+                })
```

Comment:
this map block is repeated 4 times. can it be deduped?  also, would be clearer with the tuple separate from the Http1 instantiation